.github/workflows/kicad-production.yml

```
@@ -0,0 +1,326 @@
+          echo ''
+          echo '📊 File sizes:'
+          du -sh production/*
+          "
+          
+    - name: Upload Production Files
+      uses: actions/upload-artifact@v4
+      with:
+        name: dutlink-production-files
+        path: production/
+        retention-days: 90
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)